### PR TITLE
executor , store: fix golangci-lint error when make dev

### PR DESCRIPTION
**Pull request by hecomlilong**
fix error of unnecessary use of fmt.Sprintf (gosimple) and ineffectual assignment to err (ineffassign)

### What problem does this PR solve?

Problem Summary:

### What is changed and how it works?

What's Changed:

fmt.Sprintf("select * from t") => "select * from t"
_, cluster, pdClient, err := testutils.NewMockTiKV("", nil) => _, cluster, pdClient, _ := testutils.NewMockTiKV("", nil)

How it Works:

### Related changes

- Need to cherry-pick to the release branch

Tests 

- Unit test

Side effects

- Performance regression

- Breaking backward compatibility

### Release note 

- No release note

### CODE REVIEW

**Review by ngaut — COMMENTED** [review]
LGTM

File: store/copr/coprocessor_test.go [hunk 1]
```diff
@@ -155,7 +155,7 @@ func (s *testCoprocessorSuite) TestBuildTasks(c *C) {
 func (s *testCoprocessorSuite) TestSplitRegionRanges(c *C) {
 	// nil --- 'g' --- 'n' --- 't' --- nil
 	// <-  0  -> <- 1 -> <- 2 -> <- 3 ->
-	_, cluster, pdClient, err := testutils.NewMockTiKV("", nil)
+	_, cluster, pdClient, _ := testutils.NewMockTiKV("", nil)
```
Comment (mmyj) [inline store/copr/coprocessor_test.go:158]: Should we use `c.Assert(err, IsNil)` here?

Comment (hecomlilong) [reply store/copr/coprocessor_test.go]: Yes，I think it is better.

**Review by mmyj — APPROVED** [review]
lgtm

**Comment by hecomlilong** [comment]
/cc @AilinKid 
Hi AilinKid, 
Would you please review this tiny fix for me?
Appreciate greatly.

**Comment by hecomlilong** [comment]
/cc @AilinKid @AndreMouche @Reminiscent @SunRunAway @Tjianke @TszKitLo40 @XuHuaiyu @Yisaer @b41sh @bb7133 @bobotu @breeswish @cfzjywxk @coocood @crazycs520 @fzhedu @guo-shaoge @hanfei1991 @ichn-hu @imtbkcat @jackysp @js00070 @lysu @lzmhhh123 @pingyu @qw4990 @shenli @shihongzhi @tangenta @tangwz @tiancaiamao @tsthght @winkyao @winoros @wjhuang2016 @wshwsh12 @zimulala @zz-jason
Hi all,
Would you please review this tiny fix for me?
Appreciate greatly.
It's my first time PR, so it has a particular meaning and sorry for this annoying message.

**Review by SunRunAway — APPROVED** [review]

**Pull request merged**